Kinder/LikedItems.xaml.cs

```diff
@@ -27,15 +27,31 @@ public partial class LikedItems : Window
         private string fileLocation_items = System.IO.Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName, "Data_files\\Items.txt");
         private string fileLocation_liked = System.IO.Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName, "Data_files\\Items_liked.txt");
 
+        //fileManager
+        FileManager temp2 = new();
```

Comment:
if it's temporary variable, maybe it should be placed closer to where it's used?

Reply:
I made it global so it could be used both in the constructor and elsewhere if needed

Reply:
alright